src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import java.util.*;
+
+/**
+ * The DukeException class identifies and throws Exceptions unique to Duke.
+ */
+public class DukeException extends Exception {
+    String[] validInputs = new String[] {"deadline", "todo", "event", };
+    List<String> inputList = new ArrayList<>(Arrays.asList(validInputs));
+    String[] validInputs2 = new String[] {"mark", "unmark", "delete"};
+    List<String> inputList2 = new ArrayList<>(Arrays.asList(validInputs2));
+    String[] validInputs3 = new String[] {"list", "bye"};
```

Comment:
I am not sure if it is needed for you to split all the valid inputs into 3 different `List`.